src/org/thoughtcrime/securesms/RegistrationActivity.java

```diff
@@ -447,7 +443,8 @@ private void verifyServer(TextInputEditText view) {
         String server = view.getText().toString();
         if (!TextUtils.isEmpty(server) && !Patterns.DOMAIN_NAME.matcher(server).matches()
                 && !Patterns.IP_ADDRESS.matcher(server).matches()
-                && !Patterns.WEB_URL.matcher(server).matches()) {
+                && !Patterns.WEB_URL.matcher(server).matches()
+                && !"localhost".equals(server)) {
```

Comment:
that still does not allow dotless servernames beside `localhost`, right?

probably not needed for yggmail, however, maybe we should be more relaxed here in general.

Reply:
> that still does not allow dotless servernames beside `localhost`, right?

that is only used to show a warning in the server fields, it doesn't prevent configuration, I just added "localhost" because it is a common name so the app don't warn in that case, but if you think we should not warn in any dotless domain let me know